apps/portal/src/content/docs/components/table-v2.mdx

```diff
@@ -0,0 +1,590 @@
+      </TableV2.Row>
+    </TableV2.Body>
+  </TableV2.Root>
+)
+```
+
+## Anatomy
+
+All parts of the `TableV2` component can be imported and composed as required.
+
+```typescript jsx
+<TableV2.Root>
+  <TableV2.Caption />
+  <TableV2.Header>
+    <TableV2.Row>
+      <TableV2.Head />
+    </TableV2.Row>
+  </TableV2.Header>
+  <TableV2.Body>
+    <TableV2.Row>
+      <TableV2.Cell />
+    </TableV2.Row>
+  </TableV2.Body>
+  <TableV2.Footer>
+    <TableV2.Row>
+      <TableV2.Cell />
+    </TableV2.Row>
+  </TableV2.Footer>
+</TableV2.Root>
+```
+
+## API Reference
+
+<Aside>
+  All components extend their HTML equivalents, so all relevant HTML props also
+  apply.
+</Aside>
+
+### `Root`
+
+The root component for the table.
+
+```typescript jsx
+<TableV2.Root
+  variant="default"                 // [OPTIONAL] Variant of the table
+  disableXScroll={false}            // [OPTIONAL] Disable horizontal scroll
+  className="custom-class"          // [OPTIONAL] Custom class name
+  tableClassName="custom-table-class" // [OPTIONAL] Custom table class name
+>
+</TableV2.Root>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "variant",
+      description: "Variant of the table",
+      required: false,
+      defaultValue: "default",
+      value: "'default' | 'asStackedList'",
+    },
+    {
+      name: "disableXScroll",
+      description: "Disable horizontal scroll",
+      required: false,
+      defaultValue: "false",
+      value: "boolean",
+    },
+    {
+      name: "className",
+      description: "Custom class name",
+      required: false,
+      value: "string",
+    },
+    {
+      name: "tableClassName",
+      description: "Custom table class name",
+      required: false,
+      value: "string",
+    },
+  ]}
+/>
+
+### `Header`
+
+The header section of the table.
+
+```typescript jsx
+<Table.Header
+  className="custom-class" // [OPTIONAL] Custom class name
+>
+  {/* Header content */}
+</Table.Header>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "className",
+      description: "Custom class name",
+      required: false,
+      value: "string",
+    },
+  ]}
+/>
+
+### `Body`
+
+The body section of the table.
+
+```typescript jsx
+<Table.Body
+  className="custom-class" // [OPTIONAL] Custom class name
+>
+  {/* Body content */}
+</Table.Body>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "className",
+      description: "Custom class name",
+      required: false,
+      value: "string",
+    },
+  ]}
+/>
+
+### `Footer`
+
+The footer section of the table.
+
+```typescript jsx
+<Table.Footer
+  className="custom-class" // [OPTIONAL] Custom class name
+>
+  {/* Footer content */}
+</Table.Footer>
+```
+
+<DocsPage.PropsTable
+  props={[
+    {
+      name: "className",
+      description: "Custom class name",
+      required: false,
+      value: "string",
+    },
+  ]}
+/>
+
+### `Row`
+
+A row within the table.
+
+```typescript jsx
+<Table.Row
```

Comment:
don't we have support for a `selected` prop on the row? how is row selection highlight working right now?